### ISSUE TITLE
New objects definitions for Thread network

### ISSUE THREAD

**Issue by AlexBHQ** [OpenMobileAlliance/lwm2m-registry#745]
### 1. Choose the type of registration: ###
  - [x] ```Object``` 
  - [ ] ```Reusable Resource```
  
### 2. Choose the [Object Class](http://www.openmobilealliance.org/wp/OMNA/LwM2M/LwM2MRegistry.html#registry-table): ###
  - [ ] 3rd party Standards or Alliances
  - [x] Private companies
  - [ ] Private individuals (not allowed Reusable Resources registration)
  
## 3. Confirm Software License
  - [x] Tick this box confirming that your submission has included a [software license](http://devtoolkit.openmobilealliance.org/OEditor/License).
  

Concept            | Description
:----------------|:------------------------------
Number of ```Object(s)``` or ```Reusable Resource(s)``` to register  | 4
Name of SDO, Company or individual    | Hydro-Quebec
Object Name (as defined in the .xml file) |  Thread Network, Thread Command, Thread Neighbor Information, Thread Commissioning
Short Object description, to be used in the public registry (max 134 characteres) | Objects to manage Thread network components.

**Comment by gocadimic** [comment]
Hello @AlexBHQ ,
As requested, we created a new branch [hydrtoquebec-reserve-objects](https://github.com/OpenMobileAlliance/lwm2m-registry/tree/hydrtoquebec-reserve-objects), with the 4 placeholders for the objects reservation, IDs 10483-10486.
Your next step is to fork l[wm2m-registry](https://github.com/OpenMobileAlliance/lwm2m-registry) repository and, there to prepare your objects contribution. Once it is ready, please raise a PR against the [hydrtoquebec-reserve-objects](https://github.com/OpenMobileAlliance/lwm2m-registry/tree/hydrtoquebec-reserve-objects) branch.
Once we see your PR, we will proceed with the validation.
Please let us know if you have any questions.
For more explanation, see [README.md](https://github.com/OpenMobileAlliance/lwm2m-registry#readme)

**Comment by jpradocueva** [comment]
Waiting for the company to submit the objects.

**Comment by AlexBHQ** [comment]
Hi jpradocueva
I did a pull request two weeks ago in the specified [branch](https://github.com/OpenMobileAlliance/lwm2m-registry/pull/750). 


 
Can someone accept or comment it? Do I need to create another pull request? 
I am not familiar with the process so please advise if I missed something. 
Cordially,

**Comment by jpradocueva** [comment]
@AlexBHQ, we will bring it to the DMSO group for review. All the PRs must be reviewed and agreed upon by the group before merging.

**Comment by jpradocueva** [comment]
PR #750 was agreed upon and merged by the members, adding the four (4) new objects.
